### DISCUSSION TITLE
How to get into the deep sleep for teensy 4.1

### DISCUSSION THREAD

**Discussion by WaveSimm** [micropython/micropython#15209] [m.iMX RT / Teensy 4.x] [answered]
Thanks for your advice in advance.
I am a hardware guy so, don't have enough knowledge..
Is it possible to get into deep sleep mode for teensy 4.1 using Micropython?

Thanks again.

**Comment by robert-hh** [comment]
machine.deepsleep([time_ms]) is support for the MIMXRT port including Teensy 4.1 .

**Reply by WaveSimm** [reply]
I can't get this method.  I am using 1.91
>>> machine.deepsleep(10000)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'deepsleep'

**Reply by robert-hh** [reply, marked as answer]
There is machine.deepsleep() in the current version of MicroPython.

**Reply by WaveSimm** [reply]
yes!!!!! It works well with the new version. I was using 1.91 before. 
You save my project!!!
